locales/locales.php

```diff
@@ -2050,6 +2050,16 @@ public function __construct() {
 		$rhg->slug = 'rhg';
 		$rhg->nplurals = 1;
 		$rhg->plural_expression = '0';
+		
+		$rif = new GP_Locale();
+		$rif->english_name = 'Tarifit';
+		$rif->native_name = 'Tarifiyt';
+		$rif->lang_code_iso_639_3 = 'rif';
+		$rif->country_code = 'ma';
+		$rif->wp_locale = 'rif';
```

Comment:
```suggestion
```
This should only be set for locales that are officially supported on translate.wordpress.org.